asset/test/assetTest.php

```diff
@@ -59,6 +59,20 @@ public function testExportAssets()
         $assetFile->delete();
     }
 
+    public function testListAssets()
+    {
+        $assetName = '//storage.googleapis.com/' . self::$bucketName;
+        $this->runEventuallyConsistentTest(function () use ($assetName) {
+            $output = $this->runFunctionSnippet('list_assets', [
+              'projectId' => self::$projectId,
+              'assetTypes' => ['storage.googleapis.com/Bucket'],
+              'pageSize' => 1000,
```

Comment:
use 4 spaces for tabs
```suggestion
                'projectId' => self::$projectId,
                'assetTypes' => ['storage.googleapis.com/Bucket'],
                'pageSize' => 1000,
```